includes/Vendor/VendorCache.php

```diff
@@ -17,11 +17,24 @@
 class VendorCache {
 
     public function __construct() {
-        add_action( 'dokan_new_vendor', [ $this, 'clear_cache_group' ] );
-        add_action( 'dokan_update_vendor', [ $this, 'clear_cache_group' ] );
-        add_action( 'dokan_delete_vendor', [ $this, 'clear_cache_group' ] );
-        add_action( 'dokan_vendor_enabled', [ $this, 'clear_cache_group' ] );
-        add_action( 'dokan_vendor_disabled', [ $this, 'clear_cache_group' ] );
+        add_action( 'dokan_new_vendor', [ $this, 'clear_cache_group' ], 10 );
+        add_action( 'dokan_update_vendor', [ $this, 'clear_cache_group' ], 10 );
+        add_action( 'dokan_delete_vendor', [ $this, 'clear_cache_group' ], 10 );
+        add_action( 'dokan_vendor_enabled', [ $this, 'clear_cache_group' ], 10 );
+        add_action( 'dokan_vendor_disabled', [ $this, 'clear_cache_group' ], 10 );
+        add_action( 'dokan_store_profile_saved', [ $this, 'after_update_vendor_profile' ], 10, 2 );
+        add_action( 'user_register', [ $this, 'after_created_new_wp_user' ], 10, 2 );
+    }
+
+    /**
+     * Clear vendor cache group.
+     *
+     * @since 3.3.2
+     *
+     * @return void
+     */
+    public function delete( ) {
```

Comment:
make this method static, this way whole codebase will be consistant

Reply:
ok bhai.